github/github.go

```diff
@@ -291,16 +291,18 @@ func NewClient(httpClient *http.Client) *Client {
 }
 
 // NewEnterpriseClient returns a new GitHub API client with provided
-// base URL and upload URL (often the same URL and is your GitHub Enterprise hostname).
-// If either URL does not have the suffix "/api/v3/", it will be added automatically.
+// base URL and upload URL (often is your GitHub Enterprise hostname).
+// If the base URL does not have the suffix "/api/v3/", it will be added automatically.
+// If the upload URL does not have the suffix "/api/uploads", it will be added automatically.
 // If a nil httpClient is provided, a new http.Client will be used.
 //
 // Note that NewEnterpriseClient is a convenience helper only;
 // its behavior is equivalent to using NewClient, followed by setting
 // the BaseURL and UploadURL fields.
 //
 // Another important thing is that by default, the GitHub Enterprise URL format
-// should be http(s)://[hostname]/api/v3 or you will always receive the 406 status code.
+// should be http(s)://[hostname]/api/v3/ or you will always receive the 406 status code.
+// The Upload URL format should be http(s)://[hostname]/api/uploads/.
```

Comment:
Just a nit, matching with the case for other uses of `upload URL`.
```suggestion
// The upload URL format should be http(s)://[hostname]/api/uploads/.
```